easybuild/easyconfigs/p/PLINK/PLINK-2.00a2.3LM-x86_64.eb

```diff
@@ -0,0 +1,24 @@
+# Contribution from the NIHR Biomedical Research Centre
+# Guy's and St Thomas' NHS Foundation Trust and King's College London
+# uploaded by J. Sassmannshausen
+
+easyblock = 'PackedBinary'
+
+name = 'PLINK'
+version = '2.00a2.3LM-x86_64'
+
+homepage = 'https://www.cog-genomics.org/plink/2.0/'
+description = 'Whole-genome association analysis toolset'
+
+toolchain = SYSTEM
+
+source_urls = ['https://s3.amazonaws.com/plink2-assets/alpha2']
+sources = ['plink2_linux_x86_64.zip']
```

Comment:
Problematic filename. We'll need to differentiate them locally with the download_filename approach again.

Reply:
I used a previous script as a template but I agree, I should have spotted it and changed that. I hope that is better now.